src/client/test/Integration/power-pages/actions-hub/tree-items/metadata-diff/MetadataDiffGroupTreeItem.test.ts

```diff
@@ -56,7 +56,7 @@ describe("MetadataDiffGroupTreeItem", () => {
             MetadataDiffContext.setResults(results, "Test Site", "Local Test Site", "Test Environment", "test-website-id", "test-environment-id");
             const treeItem = new MetadataDiffGroupTreeItem();
 
-            expect(treeItem.label).to.equal("Metadata Diff");
+            expect(treeItem.label).to.equal("Site Comparison");
```

Comment:
nit: Maybe we can use constant here also as it's already defined for this string.

Reply:
It's better to use actual strings in test to ensure that the value in constant file is correct. If there is some change in constant file then this test will allow us to either update the test or detect unintended change.